source/MaterialXTest/ShaderValid.cpp

```diff
@@ -16,11 +16,16 @@
 #include <MaterialXGenShader/HwShader.h>
 #include <MaterialXGenShader/HwLightHandler.h>
 
+#ifdef MATERIALX_BUILD_GEN_GLSL
```

Comment:
We didn't have proper protection when Niklas added in conditional building of generators. Adding it in now.